exchanges/binance/cfutures_types.go

```diff
@@ -376,24 +393,12 @@ type FuturesAccountInformation struct {
 		EntryPrice             float64 `json:"entryPrice,string"`
 		MaxQty                 float64 `json:"maxQty,string"`
```

Comment:
This is out of scope but would it be possible to update the assets struct with these fields as its used in coin margined futures as well: 
```go
MaxWithdrawAmount float64 `json:"maxWithdrawAmount,string"`
CrossWalletBalance float64 `json:"crossWalletBalance,string"`
CrossUnPNL float64 `json:"crossUnPnl,string"`
AvailableBalance float64 `json:"availableBalance,string"`
```